backend/hitas/models/sale.py

```diff
@@ -0,0 +1,36 @@
+from decimal import Decimal
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from hitas.models._base import ExternalHitasModel, HitasModelDecimalField
+
+
+class ApartmentSale(ExternalHitasModel):
```

Comment:
Have the filename be the same as model name? :)

Reply:
Done.